pkg/api/server/v1alpha2/results.go

```diff
@@ -67,6 +68,34 @@ func (s *Server) GetResult(ctx context.Context, req *pb.GetResultRequest) (*pb.R
 	return result.ToAPI(store), nil
 }
 
+// UpdateResult receives Result and FieldMask from client and uses them to update records in local Sqlite Server.
+func (s Server) UpdateResult(ctx context.Context, req *pb.UpdateResultRequest) (*pb.Result, error) {
+	// Retrieve result from database by name
+	tx := s.db.WithContext(ctx).Begin()
+	r, err := GetResultByName(tx, req.GetName())
+	if err != nil {
+		return nil, status.Errorf(codes.NotFound, "failed to find a result: %v", err)
+	}
+	r.Annotations = req.GetResult().GetAnnotations()
+	r.UpdatedTime = timestamppb.New(clock.Now())
+
```

Comment:
Do we want to support etag / other validation here, or want to mark it as TODO for now? (either is fine with me)

Reply:
I'm going to open another PR to support ETAG